code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -384,6 +381,7 @@
 	new /obj/item/clothing/under/det/black(src)
 	new /obj/item/clothing/under/det/classic(src)
 	new /obj/item/clothing/shoes/laceup(src)
+	new /obj/item/gun/energy/disruptorpistol/security(src)
```

Comment:
This isn't a documented change and I'm really questioning why we're giving every single security person this gun

Reply:
To be honest, I literally just forgot to update the changelog. I'm a bit stressed out over this PR, and I have a lot on my plate with all of the different people making change requests: I forgot to update this. Of course I'll update the changelog, but in the future I'd prefer it if you just let me know that I forgot to update the changelog, or messaged me directly instead of leaving a comment like this one. And if you'd like to know about the reasoning for giving nearly everyone in security this gun, we've been discussing it on and off in the code dungeon for the past two weeks, if you'd like to join in and discuss it with us. My primary impetus for giving the CSI, specifically, the pistol, is because Arrow gave me the go ahead because I agreed to also code the wireless control firing pins and control console.

Reply:
What is even wrong with this comment, I just pointed out it's undocumented and that I'm not very sure about this

Reply:
I'm at the end of my rope with this PR because I'm out of my depth. I was stressed, frustrated, and quite honestly out of my mind because I'd been working every waking hour for the last 9 days on this thing, working my real job full time and coming home just to code. I view every request made of me on the PR as a demand, and my mindset is that unless I add all of these features it'll never get merged, however as a result of all of these requests the PR has been feature creeped to the point that I'm not competent enough to implement it anymore, which is incredibly frustrating and demoralising. Your comment came at a time where I had just received a ton of extra feature requests and work that I didn't anticipate which was very stressful, and the way that I read it at the time made me feel as though it was accusatory and questioning my motives, and was going to result in the PR just being refused and all my work being for nothing. It wasn't just your comment, it was kind of just the straw on the camel's back from all the other feedback I'd received that day, especially from Karolis complaining about indentation without giving me constructive feedback about how it _should_ be which I found incredibly exasperating. I'm sorry that I misread your comment and injected my own frustrations into how I comprehended it.

Continuing on, for the reasons above I'm honestly debating internally on whether I want to even continue coding this because I feel as though nothing I do will ever be good enough, and I'm not competent enough to finish it so why even bother trying to keep up? I've been taking a break for these past few days because I'm just **tired.** A lot of this is probably miscommunication, misreading on my part or whatever, but it's still what I've internalised from the situation and it's how I feel.